.vsts-pipelines/builds/ci-internal.yml

```diff
@@ -1,13 +1,13 @@
 trigger:
-- master
+- dev
 - release/*
 
 resources:
   repositories:
   - repository: buildtools
     type: git
     name: aspnet-BuildTools
-    ref: refs/heads/release/2.2
```

Comment:
Looks like you rebased on the wrong branch. Could you fix this up?

Reply:
Oops .. I fixed